docker-app/qfieldcloud/core/models.py

```diff
@@ -1283,6 +1303,55 @@ def shared_datasets_project(self) -> Project | None:
         except Project.DoesNotExist:
             return None
 
+    def last_package_job_for_user(self, user: User) -> PackageJob | None:
```

Comment:
```suggestion
    def latest_package_job_for_user(self, user: User) -> PackageJob | None:
```
No strong opinion, I just think it is a bit more correct English?